docs/description.md

```diff
@@ -74,6 +74,8 @@ pyenv install 3.11
 pyenv local 3.11
 ```
 * Install [Poetry](https://python-poetry.org/docs/#installing-with-the-official-installer) for dependency management: 
+
+* Install a C++ compiler. See [Troubleshooting: C++ Compiler](#troubleshooting-c-compiler) for more details.
```

Comment:
Does this apply to all platforms? Or only Windows? How can a user check if already has a C++ compiler installed?

Reply:
Where is troubleshooting-c-compiler section?

Reply:
About the link, it's down below in the readme